vignettes/tic-advantages.Rmd

```diff
@@ -0,0 +1,104 @@
+---
+title: "The advantages of tic"
+author: "Patrick Schratz, Kirill Müller"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Kirill Müller}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# The advantages of tic
+
+So, what's the advantage of using `tic` over the standard CI process?
+
+## CI-Agnostic workflows
+
+What does "CI-Agnostic" mean and why do we need it?
+
+You may have heard that multiple CI services exist in the wild.
+Some are OS specific (e.g. Appveyor = Windows), some only support certain languages and for some you need to pay a monthly rate to use them.
+What all have in common: They have their own way to specify their setup file, which is most often written in YAML and stored at the top-level directory of your repo.
+This setup file controls what will be done in each run.
+
+To give you an example how different these files can be, take a look at these two examples from [Travis](https://github.com/tidyverse/dplyr/blob/master/.travis.yml) and [Appveyor](https://github.com/tidyverse/dplyr/blob/master/appveyor.yml).
+While `Travis` is the only free CI services with a stable support for R (maintained by the community, specifically by @jimhester), `Appveyor` currently requires quite some magic (e.g. `travis-tool.sh`) in the background to install a working R framework in the first place. 
+
+We could list way more differences - but that's exactly the point when `tic` comes in!
+Rather than dealing with all the CI differences, `tic` enables the specification of the complete workflow in an external R script file `tic.R`.
+The commands specified in there will be understood on every CI service that is supported by `tic`.
+
+So instead of learning how to specify one specific task on different CI platforms, the user only needs to get used to the R based approach of `tic`.
```

Comment:
More control: currently the R workflow on Travis is tied to an `.rb` file in a repo we don't control, changes are cumbersome.

Reply:
Is the official Travis implementation for R public? Could only find the old setup stuff from craig